hotham/src/shaders/pbr.frag

```diff
@@ -325,8 +325,7 @@ void main()
 				outColor.rgba = baseColor;
 				break;
 			case 2:
-				outColor.rg = (material.normalTextureID == NO_TEXTURE) ? normalize(inNormal).xy : texture(textures[material.normalTextureID], inUV).ga;
-				outColor.b = 0.0;
+				outColor.rgb = n * 0.5 + 0.5;
```

Comment:
Fantastic idea. Sasche originally intended this to preview the textures themselves but for normals this is in fact more helpful.